### PR TITLE
initial spike to set pixels on LED matrix

### PR DESCRIPTION
**Pull request by nathany**

### CODE REVIEW

**Comment by carlisia** [comment]
@nathany :+1: , I don't see anything to improve upon. I love it how you placed all of the Public methods up top. :)

File: sense/screen/color/color.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	Red   = New(255, 0, 0)
+	Green = New(0, 255, 0)
+	Blue  = New(0, 0, 255)
+	Black = New(0, 0, 0) // Off
+	White = New(255, 255, 255)
+)
+
+// New color from RGB components (0-255).
+func New(red, green, blue uint8) Color {
+	r := uint16(red>>3) & 0x1F
```
Comment (nathany) [inline sense/screen/color/color.go]: I don't the masking is necessary because these are uint8 already.

File: examples/screen/main.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import (
+	"time"
+
+	"github.com/gophergala2016/bobblehat/sense/screen"
+	"github.com/gophergala2016/bobblehat/sense/screen/color"
+)
+
+func main() {
+	var frontBuffer screen.FrameBuffer
```
Comment (nathany) [inline examples/screen/main.go]: maybe just var fb

File: sense/screen/screen.go [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+func init() {
+	var err error
+	displayDevice, err = getDevice("RPi-Sense FB")
+	if err != nil {
+		panic(err)
+	}
+}
+
+// Draw a buffer to the LED matrix screen.
+func Draw(fb *FrameBuffer) error {
```
Comment (nathany) [inline sense/screen/screen.go]: move up

**Pull request merged**